GVRf/Framework/framework/src/main/java/org/gearvrf/GVRBitmapTexture.java

```diff
@@ -268,6 +268,8 @@ public Boolean call() {
     private boolean updateCall(Bitmap bitmap) {
         glBindTexture(GL_TEXTURE_2D, getId());
         GLUtils.texImage2D(GL_TEXTURE_2D, 0, bitmap, 0);
+        android.opengl.GLES20.glGenerateMipmap(GL_TEXTURE_2D);
```

Comment:
Minor coding style comment. Please import the statics from the GLES20 package, and reduce this line to glGenerateMapmap(...) or GLES20.gl*.